src/TConsole.cpp

```diff
@@ -2681,6 +2682,42 @@ bool TConsole::setBackgroundColor(const QString & name, int r, int g, int b, int
 
 }
 
+bool TConsole::raiseWindow(const QString & name )
+{
+    std::string key = name.toLatin1().data();
```

Comment:
Unfortunately, mSubConsoleMap is a std::map, not a QMap. std::map does not have a contains() method, so I think we're stuck using mSubConsoleMap.find() until such a time as we redo everything based on using a QMap instead.